crc32c/CMakeLists.txt

```diff
@@ -1,5 +1,15 @@
+set(crc32c_include_directories ${PROJECT_SOURCE_DIR}/util)
+
+# Build the crc32c implementation as a library to be linked into the database server.
 add_library(crc32c crc32c.c)
-include_directories(${PROJECT_SOURCE_DIR}/util)
+target_include_directories(crc32c PRIVATE ${crc32c_include_directories})
+
+# Build the crc32c test executable with UBSAN.
+add_executable(crc32c_test crc32c.c)
```

Comment:
This should go under `tests/tools`

Reply:
Are you suggesting moving the tests into a separate `.c` file under `test/tools` and removing them from `crc32c.c` entirely?

Reply:
That would work as well, and probably it would be better that way. I was initially thinking moving just CMake portion.